astrokat/test/test_offline_observe.py

```diff
@@ -36,18 +40,25 @@ def test_two_calib_sim(self):
         cal1 = result.count("0408-65 observed for 30.0 sec")
         cal2 = result.count("1934-638 observed for 30.0 sec")
 
-        self.assertGreaterEqual(cal1 + cal2, 1, "At least one bpcal was observed")
+        self.assertGreaterEqual(
+            cal1 + cal2, 1, "At least one bpcal was observed")
         self.assertLessEqual(cal1 + cal2, 2, "At most 2 bpcals were observed")
```

Comment:
To fix this error I would need to change the implementation of the test.
`"/home/kat/workspace/946_linting_with_flake8_astrokat/astrokat/test/test_offline_observe.py", line 45, in test_two_calib_sim`

`self.assertLessEqual(cal1 + cal2, 2, "At most 2 bpcals were observed"
AssertionError: 12 not less than or equal to 2 : At most 2 bpcals were observed
"""Fail immediately, with the given message."""
[2019-09-05T10:51:04.076Z] >>  raise self.failureException('12 not less than or equal to 2 : At most 2 bpcals were observed')`